packages/keystatic/package.json

```diff
@@ -284,6 +284,18 @@
     "#sha1": {
       "node": "./src/sha1/node.ts",
       "default": "./src/sha1/webcrypto.ts"
+    },
+    "#field-ui/*": {
+      "react-server": "./src/form/fields/empty-field-ui.tsx",
+      "default": "./src/form/fields/*/ui.tsx"
+    },
+    "#component-block-primitives": {
+      "react-server": "./src/form/fields/document/DocumentEditor/primitives/blank-for-react-server.tsx",
+      "default": "./src/form/fields/document/DocumentEditor/primitives/index.tsx"
+    },
+    "#cloud-image-preview": {
```

Comment:
this feels ... moderately unsustainable, if we end up with many more component blocks? could/should we organise this differently, like have an `component-blocks` import? or is this just private API and it doesn't matter, we can refactor it later if that's an issue?

Reply:
> is this just private API and it doesn't matter, we can refactor it later if that's an issue?

This, all of the `imports` field is entirely private